server/routerlicious/packages/services/src/secretManager.ts

```diff
@@ -0,0 +1,21 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import * as core from "@fluidframework/server-services-core";
+
+/**
+ * Manages encryption of secrets.
+ */
+export class SecretManager  implements core.ISecretManager {
+    public decryptSecret(encryptedSecret: string): string {
+        /** Add your custom implementation here. */
```

Comment:
Rather than saying "add your implementation", I would say that this is a dummy implementation which returns the same passed key after encryption and decryption. Users requiring encryption are expected to have their own implementation of ISecretManager

Reply:
Makes sense. done.:)